whylog/tests/tests_log_reader/test_read.py

```diff
@@ -9,19 +12,50 @@
 class TestLogsReading(TestCase):
     @classmethod
     def setUpClass(cls):
+        cls.time_delta_ms = 100
+        cls.number_of_lines = 10000
+        cls.line_padding = 42
         cls.opened_file = OperationCountingFileWrapper(
             DataGeneratorLogSource(
-                datetime.datetime(
+                start_time=datetime.datetime(
                     year=2000,
                     month=1,
                     day=1
                 ),
-                datetime.timedelta(milliseconds=1),
-                10000,
-                42,
-                "%c"
+                time_delta=datetime.timedelta(milliseconds=cls.time_delta_ms),
+                number_of_lines=cls.number_of_lines,
+                line_padding=cls.line_padding,
+                datetime_format="%c"
             )
         )  # yapf: disable
+        cls.the_earliest_date = datetime.datetime(year=1, month=1, day=1)
```

Comment:
that's just `datetime.max` and `datetime.min`, constants which are already defined. Use them inline.